com.spotify.Client.json

```diff
@@ -105,6 +105,21 @@
                 }
             ]
         },
+        {
+            "name": "spotifywm",
+            "buildsystem": "simple",
+            "build-commands": [
+                "make",
+                "install -Dm644 spotifywm.so /app/lib/spotifywm.so"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/dasJ/spotifywm.git",
+                    "commit": "91dd553"
```

Comment:
Why not full commit? `91dd5532ffb7a398d775abe94fe7781904ab406f`

Reply:
I'm not opposed to that modification but I also don't really want to clutter up the revision history. I am 99.99999% sure there won't be a subsequent commit that collides with the first seven hex digits. I'll do it if y'all really want me to, I just want to make sure. :)

Reply:
Revision history doesn't matter as @TingPing may just squash all commits here as they don't add any value. Similarly to https://github.com/flathub/com.spotify.Client/pull/142#discussion_r512419204 why make assumptions about something can or can't happen if doing the right things is costless? I would say using short commits is pretty unusual thing to do.

Reply:
Okay, sounds good to me! I'll push up the change in a few moments :)